src/players.ts

```diff
@@ -0,0 +1,500 @@
+const allPlayers = Object.fromEntries(`√ìculo,19
```

Comment:
We should stick this into a site/API and load it from there instead of bundling into the extension.